osc-collab.py

```diff
@@ -4062,3 +4063,14 @@ def do_collab(self, subcmd, opts, *args):
 
     else:
         raise RuntimeError('Unknown command: %s' % cmd)
+
+def identify_conf():
+    # needed for compat reasons(users may have their oscrc still in ~
+    if 'OSC_CONFIG' in os.environ:
+        return os.environ.get('OSC_CONFIG')
```

Comment:
nit: instead of returning, use the `conffile` that you return at the end, and use `elif`

The rest LGTM

Reply:
Since it's the same code as the one in osc, and I guess I'll leave it that way.